src/Components/Forms/DiagnosisAndRecommendations.js

```diff
@@ -0,0 +1,26 @@
+//DiagnosisAndRecommendations
+
+import { connect } from 'react-redux';
+import React from 'react';
+
+const DiagnosisAndRecommendations = ({
+  patient,
+  encounter,
+  formatDate,
+  languageDirection,
+  history,
+  verticalName,
+}) => {
+  return <React.Fragment>DiagnosisAndRecommendations</React.Fragment>;
+};
+
+const mapStateToProps = (state) => {
+  return {
```

Comment:
missing in the props the permission mode

Reply:
not relevent